lib/node_modules/@stdlib/math/base/complex/divide/README.md

```diff
@@ -0,0 +1,96 @@
+# divide
+
+> Divide two complex numbers.
+
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var cdiv = require( '@stdlib/math/base/complex/divide' );
+```
+
+#### cdiv( \[quotient,\] re1, im1, re2, im2 )
+
+Divides two `complex` numbers where each `complex` number is comprised of a __real__ component `re` and an __imaginary__ component `im`. If `quotient` array is provided, result will be written in-place and the reference returned. If `quotient` is not specified, an array containing the quotient is allocated and returned.
```

Comment:
Move sentence `If quotient is provided...` to below as shown in follow-up comment.